ddtrace/contrib/langchain/patch.py

```diff
@@ -675,6 +675,72 @@ async def traced_chain_acall(langchain, pin, func, instance, args, kwargs):
     return final_outputs
 
 
+@with_traced_module
+def traced_lcecl_runnable_sequence(langchain, pin, func, instance, args, kwargs):
```

Comment:
```suggestion
def traced_lcel_runnable_sequence(langchain, pin, func, instance, args, kwargs):
```

Reply:
Can we add a small comment explaining what a lcel is (to give context for future reviewers who might not know that a lcel is just a implicit chain?)